analyzers/src/SonarAnalyzer.CSharp/Rules/NonDerivedPrivateClassesShouldBeSealed.cs

```diff
@@ -24,48 +24,74 @@ namespace SonarAnalyzer.Rules.CSharp
     public sealed class NonDerivedPrivateClassesShouldBeSealed : SonarDiagnosticAnalyzer
     {
         private const string DiagnosticId = "S3260";
-        private const string MessageFormat = "Private classes or records which are not derived in the current assembly should be marked as 'sealed'.";
+        private const string MessageFormat = "Private or file classes or records which are not derived in the current assembly should be marked as 'sealed'.";
```

Comment:
I don't really like the generic text on MessageFormat, It's not a strong opinion but we might also improve it by passing as `messageArgs` the precise cause of the issue.
```suggestion
        private const string MessageFormat = "'{0}' which are not derived in the current assembly should be marked as 'sealed'.";
```
Passing either:

- Private classes
- File classes
- Private records
- File records

Reply:
Good suggestion! Implemented it.